CHANGES

```diff
@@ -1,6 +1,15 @@
 Changes
 =======
 
+Version 0.3.0 (released 2015-03-16)
+
+- New method `has_active_child(recursive=True)` in `MenuEntryMixin`.  (#25)
+- Fixed documentation of blueprint example.
+- Initial configuration for Docker and demo app.
+- Fixed template example and added code block types.  (#14)
+- NOTE The upper limit on the version of `coverage` has been  set since the
+  latest one was found to be failing.
```

Comment:
- The note has two spaces after "has been", but I'd remove this "coverage" point completely, since it is of internal testing use only.
- "Initial ...": let's improve that Docker configuration before releasing... I'll send a separate PR.